include/LightGBM/config.h

```diff
@@ -329,6 +330,25 @@ struct Config {
   // desc = **Note**: if balanced bagging is enabled, ``bagging_fraction`` will be ignored
   double neg_bagging_fraction = 1.0;
 
+  // default = 1e-4
+  // check = >0.0
+  // desc = used in MVS boosting. If ``mvs_adaptive == true`` then this value is ignored.
+  // desc = used only in ``mvs``
+  double mvs_lambda = 1e-4;
+
+  // default = false
+  // desc = use adaptive variant of mvs boosting
+  // desc = used only in ``mvs``
+  bool mvs_adaptive = false;
+
+  // default = 256000
+  // check = >0
+  // desc = used in MVS boosting training. If dataset size is greater than ``mvs_max_sequential_size``, then threshold
+  // desc = for MVS is chosen for each thread independently.
+  // desc = used only in ``mvs``
+  // desc = **Note**: on small dataset setting this parameter less than size of dataset may produce results depending on number of threads
```

Comment:
```suggestion
  // desc = **Note**: on small dataset setting this parameter less than size of dataset may produce non-deterministic results depending on number of threads
```

I'm not sure what this statement was supposed to be, but I'm fairly sure it wasn't just "may produce results". Is this not about the results being non-deterministic? Or should it maybe be "bad" results?

Reply:
I think it should be non-deterministic. Thanks for the fix.